docs/glossary.md

```diff
@@ -4,80 +4,102 @@ slug: /reference/glossary
 ---
 
 ### Glossary
+
 ### `Eigenlayer`
+
 An Ethereum-based protocol that enables restaking
 
 ### `AVS`
+
 Actively Validated Service ~ the term for a service on Eigenlayer
 
 ### `Restaking`
+
 Where people reuse collateral to operate ≥ 1 AVS. If this collateral is beacon chain ETH, they are a called a `native restaker`
 
 ### `LSP`
+
 Liquid Staking Protocol
 
 ### `nLRP`
+
 Native Liquid Restaking Protocol. The Puffer protocol offers native ETH liquid restaking on Eigenlayer
 
 ### `pufETH`
+
 The yield bearing token that represents staked ETH in Puffer’s LSP
 
 ### `Stakers`
+
 The users who “stake” ETH to the vault to receive pufETH
 
 ### `NoOps`
+
 The Puffer nodes that operate an Ethereum validator (and optionally additional AVSs)
 
 ### `Guardians`
+
 A committee of Ethereum aligned community members and organizations to assist the protocol as it matures
 
 ### `Enclave`
+
 A secure-computing environment capable of running Puffer’s anti-slashing technology
 
 ### `Burst Threshold`
+
 Code in our smart contracts that self-caps the vault at 22% marketshare
 
 ### `Beacon Chain`
+
 The main component of the Ethereum blockchain responsible for Proof of Stake consensus
 
 ### `Beacon Chain Deposit Contract`
+
 The smart contract to which a NoOp must deposit 32 ETH to initiate running a validator node
 
 ### `VEM`
+
 Voluntary Exit Message ~ a validator-signed message that will exit the signer when broadcasted to the beacon chain
 
 ### `Withdrawal Credentials`
+
 A validator parameter that sets where their consensus rewards and full withdrawal eth are sent. For 0x01 (post-Shapella) withdrawals, the credentials are formatted as `0x01000000000000` ++ `20 byte ethereum address`
```

Comment:
Maybe add  **0x02** for Pectra validators `0x02000000000000` ++ `20 byte ethereum address`